src/clients/c++/perf_client/perf_client.cc

```diff
@@ -259,142 +275,293 @@ Usage(char** argv, const std::string& msg = std::string())
-  std::cerr << "For -r, it indicates the maximum number of measurements for "
-               "each profiling setting. The perf client will take multiple "
-               "measurements and report the measurement until it is stable. "
-               "The perf client will abort if the measurement is still "
-               "unstable after the maximum number of measuremnts."
+      << FormatMessage(
+             " --percentile: It indicates that the specified percentile in "
+             "terms of latency will also be reported and used to detemine if "
+             "the measurement is stable instead of average latency. Default "
+             "is -1 to indicate no percentile will be used.",
+             18)
+      << std::endl;
+
+  std::cerr << "\tDEPRECATED OPTIONS" << std::endl;
+  std::cerr << std::setw(9) << std::left << " -t: "
+            << FormatMessage(
+                   "It indicates the number of starting concurrent requests if "
+                   "-d flag is set.",
+                   9)
             << std::endl;
-  std::cerr << "For -l, it has no effect unless -d flag is set." << std::endl;
   std::cerr
-      << "If -x is not specified the most recent version (that is, the highest "
-      << "numbered version) of the model will be used." << std::endl;
-  std::cerr << "For -i, available protocols are gRPC and HTTP. Default is HTTP."
+      << std::setw(9) << std::left << " -c: "
+      << FormatMessage(
+             "It indicates the maximum number of concurrent requests allowed "
+             "if -d flag is set. Once the number of concurrent requests "
+             "exceeds the maximum, the perf client will stop and exit "
+             "regardless of the latency threshold. Default is 0 to indicate "
+             "that no limit is set on the number of concurrent requests.",
+             9)
+      << std::endl;
+  std::cerr
+      << std::setw(9) << std::left << " -d: "
+      << FormatMessage(
+             "This flag enables dynamic concurrent request count where the "
+             "number of concurrent requests will increase linearly until the "
+             "request latency is above the threshold set (see -l).",
+             9)
+      << std::endl;
+  std::cerr << std::setw(9) << std::left << " -a: "
+            << FormatMessage(
+                   "This flag is deprecated and will not affect client.", 9)
+            << std::endl;
+
+  std::cerr << std::endl;
+  std::cerr << "II. INPUT DATA OPTIONS: " << std::endl;
+  std::cerr << std::setw(9) << std::left
+            << " -b: " << FormatMessage("Batch size for each request sent.", 9)
+            << std::endl;
+  std::cerr << FormatMessage(
+                   " --input-data: This option can be used to select data, "
+                   "client will use in it's inference requests. The available "
+                   "options are \"zero\", \"random\" or path to a directory. "
+                   "If the option is path to a directory then there must be a "
```

Comment:
"The directory must contain a binary file for each input, named the same as the input. Each file must contain the data required for that input for a batch-1 request. Each file should contain the raw binary representation of the input in row-major order. Default is \"random\"."